project-base/app/translations/messages.en.po

```diff
@@ -25,6 +25,9 @@ msgstr ""
 msgid "Administrator"
 msgstr ""
 
+msgid "Alt"
+msgstr ""
```

Comment:
To confirm, is it acceptable to have empty strings? If so, that should be fine.

Reply:
All lines are empty. So I think it is ok